### PR TITLE
feat: upload video button and new video thumbnail

### PR DESCRIPTION
**Pull request by WaDadidou**
Add upload video button in social feed's video tab

button: https://www.figma.com/file/WLEMAQXohIwBEPL5lsRrvK/Teritori---dApp?type=design&node-id=14682-287916&mode=design&t=CsyFyilB8z6wJh0z-4

flow: https://www.figma.com/file/WLEMAQXohIwBEPL5lsRrvK/Teritori---dApp?type=design&node-id=14683-289637&mode=design&t=CsyFyilB8z6wJh0z-4

And rework video thumbnail to use the new description field like in https://www.figma.com/file/WLEMAQXohIwBEPL5lsRrvK/Teritori---dApp?type=design&node-id=20319-162643&mode=design&t=CsyFyilB8z6wJh0z-4
___

Video upload modal

image

Video in Jungle Feed

image

Videos posted as usualy (Through NewsFeedInput) have now the VideoNote PostCategory
image

X - No duration because `onloadedmetadata` is not triggered here : https://github.com/TERITORI/teritori-dapp/blob/1130a8ad5718fc7ced4597b323322616e4c09052/packages/utils/video/index.ts#L9

X - Have to generate thumbnail image and use it as thumbnail on Video upload if no image provided by the user.
Here : https://github.com/TERITORI/teritori-dapp/blob/1130a8ad5718fc7ced4597b323322616e4c09052/packages/utils/video/index.ts#L24-L23

### CODE REVIEW

File: packages/components/video/VideoCard.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+  if (!video) return null;
+  return (
+    <View style={unitCardStyle}>
+      <CustomPressable
+        style={imgBoxStyle}
+        onHoverIn={() => setIsHovered(true)}
+        onHoverOut={() => setIsHovered(false)}
+        // TODO:
+        // onPress={() => {
```
Comment (omniwired) [inline packages/components/video/VideoCard.tsx:63]: is this todo forgotten?

File: assets/default-images/default-video-thumbnail.png
Comment (omniwired) [inline assets/default-images/default-video-thumbnail.png:1]: please optimize this one using https://squoosh.app/

File: packages/components/video/VideoCard.tsx (hunk 1 above)
Comment (WaDadidou) [reply packages/components/video/VideoCard.tsx:63]: No, FeedVideoViewScreen will be made in another PR

File: assets/default-images/default-video-thumbnail.png
Comment (WaDadidou) [reply assets/default-images/default-video-thumbnail.png:1]: Done

**Pull request merged**